### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by tahnik**

### CODE REVIEW

**Comment by tahnik** [comment]
#107

**Review by hex2f — APPROVED** [review]

File: app/src/main/js/components/item/bottom_bar.js [hunk 1]
```diff
@@ -48,15 +50,13 @@ class BottomBar extends Component {
   render() {
     const { comments, type } = this.props;
     const disabled = this.props.isUser ? 'disabled' : '';
+    const upvoted = this.props.isUser ? 'disabled' : '';
     return (
       <div className="bottom_bar_container" >
         <div
-          className={`upvote ${disabled}`}
+          className={`upvote ${disabled} ${this.state.isUpvoted > 0 ? 'upvoted' : ''}`}
```
Comment (tahnik) [inline app/src/main/js/components/item/bottom_bar.js:56]: @RekkyRek what is happening in here?

Comment (hex2f) [reply app/src/main/js/components/item/bottom_bar.js:56]: It adds a class if voted to control the ::before element.

Comment (tahnik) [reply app/src/main/js/components/item/bottom_bar.js:56]: I was just wondering why you didn't use the variable on line 53

Comment (tahnik) [reply app/src/main/js/components/item/bottom_bar.js:56]: nvm I will removed that

**Pull request merged**